packages/windows/data_stream/sysmon_operational/elasticsearch/ingest_pipeline/default.yml

```
@@ -256,26 +256,29 @@ processors:
       target_field: error.code
       ignore_failure: true
       ignore_missing: true
-      if: ctx.event.code == "255"
+      if: ctx.event.code == "255" && ctx.winlog?.event_data?.ID != null && ctx.winlog?.event_data?.ID != ""
```

```suggestion
      if: ctx.event.code == "255" && ctx.winlog?.event_data?.ID != null && ctx.winlog.event_data.ID != ""
```

Same for the other `if`s in this pipeline.

---

Same above.